ob_intern/Controller/Create Short Video/CreateShortVideoViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  CreateShortVideoViewController.swift
+//  ob_intern
+//
+//  Created by Soft Liampisan on 8/6/2566 BE.
+//  Copyright (c) 2566 BE ___ORGANIZATIONNAME___. All rights reserved.
+
+import UIKit
+
+class CreateShortVideoViewController: UIViewController {
+
+    @IBOutlet weak var textViewThoughts: UITextView!
+    @IBOutlet weak var imageViewVideo: UIImageView!
+    
+    //MARK: - New Instance
+    class func newInstance() -> CreateShortVideoViewController {
+        let viewController = CreateShortVideoViewController(nibName: String(describing: CreateShortVideoViewController.self),
+                                                       bundle: nil)
+        
+        
+        let viewModel = CreateShortVideoViewModel(delegate: viewController)
+        viewController.viewModel = viewModel
+        
+        return viewController
+    }
+    
+    //MARK: - IBOutlet
+    
+    //MARK: - Parameters
+    private var viewModel: CreateShortVideoViewModel?
+    
+    //MARK: - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.setupView()
+    }
+    
+    //MARK: - Functions
+    func setupView() {
+        textViewThoughts.delegate = self
+        textViewThoughts.text = "Share your thoughts within 2000 characters"
```

Comment:
มันจะมี placeholder ของมันเอง

Reply:
ลองหาแล้วเขาเขียนว่า UITextView doesn't inherently have a placeholder property คะ
[https://stackoverflow.com/questions/27652227/add-placeholder-text-inside-uitextview-in-swift#:~:text=UITextView%20doesn't%20inherently%20have,the%20class%20and%20set%20textView.](url)